client/src/utils/saveload.ts

```diff
@@ -35,28 +34,44 @@ export function useStoreInitialState(tableId: string, state: TableState) {
     if (isLocalStorageAvailable) {
       localStorage.setItem(`${tableId}-sorters`, JSON.stringify(state.sorters));
     }
+    if (JSON.stringify(state.sorters) != JSON.stringify([{ field: "id", order: "asc" }])) {
+      updateURLHash(`${tableId}-sorters`, JSON.stringify(state.sorters));
```

Comment:
I'm wondering if it's necessary to include the tableId in the hash, since there will only be a single table per page anyway. The page URL itself will kinda decide the tableId.

Reply:
You're probably right about that. I was just keeping the keys the same between localStorage and the hash. Excluding it would shorten the hash a little bit and it should be unnecessary.